aptos-move/framework/aptos-framework/sources/object.spec.move

```diff
@@ -1,9 +1,6 @@
 spec aptos_framework::object {
-    spec exists_at {
-        pragma opaque;
-        aborts_if false;
-        // TODO: Disabled the following post-condition due to an issue with
-        // the use of a type parameter in `exists` in spec.
-        // ensures result == exists<T>(object);
+    spec exists_at<T>(object: address): bool {
+        pragma intrinsic;
     }
+    spec native fun spec_exists_at<T>(object: address): bool;
```

Comment:
This does not have to be native.

If this function is really needed, you can defined it as the following:

```move
spec fun spec_exists_at<T>(object: address): bool {
    exists<T>(object)
}
```

Otherwise, simply omit this one and use `exists<T>(addr)` in spec instead.

Reply:
Right. Then, we're hitting the original issue that we can't use a type parameter `T` in `exists` both in spec and code: https://github.com/aptos-labs/aptos-core/issues/6658

Perhaps, should Prover either support a type parameter `T` in `exists` or give a special treatment of `spec_exists_at`?